Sources/App/Models/Package.swift

```diff
@@ -42,7 +42,7 @@ final class Package: Model, Content {
     @OptionalEnum(key: "processing_stage")
     var processingStage: ProcessingStage?
 
-    @Field(key: "last_commit_at")  // TODO: shouldn't this rather live in Repository?
+    @Field(key: "last_commit_at")  // TODO: shouldn't this rather live in Repository? Is it needed at all?
     var lastCommitAt: Date?
```

Comment:
Do we need this field at all? Or was this something the previous version used for processing stage management, which we're doing via `processing_stage` now?